### PR TITLE
Fix products categories comments replycomments collections

### PR DESCRIPTION
**Pull request by vhazna**
Title: Improving tests and fixing bugs in products, categories, comments, and replycomments collections.

Description: In our demo meeting on Friday (23.09), I've found some bugs in my code. This PR is about improving those bugs and also I worked on improving tests coverage to 100% 

Ticket link: there's no ticket about this.

### CODE REVIEW

File: src/middlewares/validators/createCommentsPayload.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const Joi = require('joi');
 
 const createCommentsPayloadValidator = (req, res, next) => {
   const schema = Joi.object({
-    comment: Joi.string().min(2).max(30).required(),
+    comment: Joi.string().min(2).required(),
```
Comment (VasylLymych) [inline src/middlewares/validators/createCommentsPayload.js]: Set min to be 1. There must not be comment length limits . It must be just not empty

File: src/middlewares/validators/createReplyCommentsPayload.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ const Joi = require('joi');
 
 const createReplyCommentsPayloadValidator = (req, res, next) => {
   const schema = Joi.object({
-    comment: Joi.string().min(2).max(30).required(),
+    comment: Joi.string().min(2).required(),
```
Comment (VasylLymych) [inline src/middlewares/validators/createReplyCommentsPayload.js]: Same as above

File: src/middlewares/validators/updateCommentsPayload.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ const Joi = require('joi');
 
 const updateCommentsPayloadValidator = (req, res, next) => {
   const schema = Joi.object({
-    comment: Joi.string().min(2).max(30).required()
+    comment: Joi.string().min(2).required()
```
Comment (VasylLymych) [inline src/middlewares/validators/updateCommentsPayload.js]: Same here

**Review by VasylLymych — APPROVED** [review]

File: src/middlewares/validators/updateCommentsPayload.js (hunk 3 above)
Comment (vhazna) [reply src/middlewares/validators/updateCommentsPayload.js]: Fixed.

File: src/middlewares/validators/createReplyCommentsPayload.js (hunk 2 above)
Comment (vhazna) [reply src/middlewares/validators/createReplyCommentsPayload.js]: Fixed.

File: src/middlewares/validators/createCommentsPayload.js (hunk 1 above)
Comment (vhazna) [reply src/middlewares/validators/createCommentsPayload.js]: Fixed.

**Pull request merged**